src/video/wayland/SDL_waylandwindow.c

```diff
@@ -775,6 +775,19 @@ void Wayland_ShowWindow(_THIS, SDL_Window *window)
     if (window->flags & SDL_WINDOW_BORDERLESS) {
         Wayland_SetWindowBordered(_this, window, SDL_FALSE);
     }
+
+    /* We're finally done putting the window together, raise if possible */
+    if (c->activation_manager) {
+        const char *activation_token = SDL_getenv("XDG_ACTIVATION_TOKEN");
+        if (activation_token && *activation_token != '\0') {
+            xdg_activation_v1_activate(c->activation_manager,
+                                       activation_token,
+                                       data->surface);
+
+            /* Clear this variable, per the protocol's request */
+            SDL_setenv("XDG_ACTIVATION_TOKEN", "", 1);
```

Comment:
Is there no "unsetenv"? I assume this just sets it to an empty string, which might confuse any application inheriting this process's environment variables. An empty string is technically a "valid" token, so in general apps shouldn't try to introspect it.

Reply:
There is no SDL_unsetenv as far as I know - I suppose we could just call unsetenv directly within this subsystem, unless we want to be strict about this. @icculus, thoughts about this call?

Reply:
Nevermind, I'm a dummy, we actually _have_ unsetenv built into our stdlib:

https://github.com/libsdl-org/SDL/blob/main/src/stdlib/SDL_getenv.c#L91

Update incoming.